prod-refractr.yml

```diff
@@ -828,3 +828,9 @@ refracts:
   - https://bugzil.la/123456: https://bugzilla.mozilla.org/show_bug.cgi?id=123456
   - https://bugzil.la/form.foo: https://bugzilla.mozilla.org/form.foo
   - https://bugzil.la/glonk: https://bugzilla.mozilla.org/buglist.cgi?quicksearch=glonk
+
+# https://mozilla-hub.atlassian.net/browse/SVCSE-140
+- status.relay.firefox.com/: https://www.mozilla.org/
+  status: 302
+- status.vpn.mozilla.org/: https://www.mozilla.org/
```

Comment:
```suggestion
- www.mozilla.org/: status.vpn.mozilla.org
```

Reply:
i think you want status.vpn.mozilla.org to go to www.mozilla.org, yeah? (http to https is included for free). ditto with guy above. also, yeah, i know, its backwards in refractr spec land.

Reply:
my bad, thank you!